src/features/projectsV2/ProjectDetails/components/KeyInfo.tsx

```diff
@@ -0,0 +1,149 @@
+import { Fragment } from 'react';
+import styles from '../styles/ProjectInfo.module.scss';
+import { useTranslations } from 'next-intl';
+import formatDate from '../../../../utils/countryCurrency/getFormattedDate';
+import SingleProjectInfoItem from './microComponents/SingleProjectInfoItem';
+import InfoIconPopup from '../../../../temp/components/InfoIconPopup';
+import InterventionSeason from './microComponents/InterventionSeason';
+import { AllowedSeasonMonths } from '@planet-sdk/common';
+
+interface Props {
+  abandonment: number | null;
+  interventionStarted: string | number | null;
+  plantingDensity: number | null;
+  maxPlantingDensity: number | null;
+  employees: number | null;
+  interventionSeasons: number[] | AllowedSeasonMonths[] | null;
+  isTreeProject: boolean;
+  degradationYear: number | null;
+}
+
+const KeyInfo = ({
+  abandonment,
+  interventionStarted,
+  plantingDensity,
+  maxPlantingDensity,
+  employees,
+  interventionSeasons,
+  isTreeProject,
+  degradationYear,
+}: Props) => {
+  const tCommon = useTranslations('Common');
+  const tManageProjects = useTranslations('ManageProjects');
+  const tProjectDetails = useTranslations('ProjectDetails');
+
+  const addZeroToDate = (val: string) => {
+    const arr = val.split('-');
+    const newDateArr = [arr[0]];
+    if (arr[1].length === 1) {
+      newDateArr.push(`0${arr[1]}`);
+    } else {
+      newDateArr.push(arr[1]);
+    }
+    if (arr[2].length === 1) {
+      newDateArr.push(`0${arr[2]}`);
+    } else {
+      newDateArr.push(arr[2]);
+    }
+    return newDateArr.join('-');
+  };
+  const shouldRenderInterventionDate =
+    (interventionStarted &&
+      typeof interventionStarted === 'string' &&
+      interventionStarted?.length > 0) ||
+    interventionStarted;
+
+  return (
+    <div className={styles.projectInfoContainer}>
+      <div className={styles.singleRowInfoContainer}>
```

Comment:
Minor rename - could this be `keyInfoContainer` instead of `projectInfoContainer` - both because it contains the `keyInfo` and also because `projectInfoContainer` and `projectInfoSection` are similar names and could cause confusion.